packages/ui-components/src/features/Auth/AuthViewButton.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import styled from 'styled-components';
+import { Button } from '../../components';
+
+export const AuthViewButton = styled(Button)`
+  text-transform: none;
+  font-size: 1rem;
+  width: 22rem;
+  max-width: 100%;
+  margin-left: 0 !important;
```

Comment:
Does it actually need important? Can we just use a couple of MUI classes instead?